### DISCUSSION TITLE
Alarm priority on Wear OS 3.0 / Galaxy watch 4

### DISCUSSION THREAD

**Discussion by AlexD97** [Navid200/xDrip#80] [Watches]
On Wear OS 2.0 xDrip alarms made my watch vibrate intensively 3 times. On Wear OS 3.0 on the Galaxy watch 4 the watch vibrates like for ordinary notifications. The problem is that I may not notice the alarm.
Is it possible to adjust the intensity of the vibration for the alarm notification on Wear OS 3.0?

**Comment by Navid200** [comment]
Isn't that just a function of the watch itself?
I mean are you suggesting that the same watch behaves differently under two different xDrip settings?
Or, do you mean that one watch vibrates intensively while another watch doesn't?

**Comment by AlexD97** [comment]
Previously I had a Fossil Gen 3 (Wear OS 2) smartwatch. There I got 3 intense vibrations when xDrip had an alarm (other notifications like Whatsapp made only slight vibrations). Now on the Galaxy watch 4 (Wear OS 3) the xDrip alarm notification behaves like every other notification (only a slight vibration which is easy to miss). 
I hope I could clearify a little bit...

**Comment by Navid200** [comment]
There is an existing request to let user change the vibration pattern for a handset:
https://github.com/NightscoutFoundation/xDrip/issues/1448

I'm not suggesting that your request is a duplicate.  But, I think it is closely related.
If you agree, would you mind adding your request to that issue also?  Please emphasize that you are requesting it for a watch.

**Reply by AlexD97** [reply]
Thanks, I added this to [#1448](https://github.com/NightscoutFoundation/xDrip/issues/1448)
